src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -0,0 +1,177 @@
+  const [currentApplication, setCurrentApplication] = useState({});
+
+  const handleOk = () => {
+    setDisplayModal(false);
+  };
+
+  const handleCancel = () => {
+    setDisplayModal(false);
+    setProfileId('');
+  };
+
+  useEffect(() => {
+    const getCurrentApp = () => {
+      axiosWithAuth()
+        .get(`/application/profileId/${profileId}`)
+        .then(res => {
+          setCurrentApplication(res.data);
+        })
+        .catch(err => {
+          console.log(err);
+        });
+    };
+    getCurrentApp();
+  }, [profileId]);
+
+  return (
+    <>
+      {currentApplication.role_name === undefined ? (
+        <Modal
+          visible={displayModal}
+          onOk={handleOk}
+          onCancel={handleCancel}
+          afterClose={handleCancel}
+          footer={null}
+        >
+          Application not found
+        </Modal>
+      ) : (
+        <Modal
+          title="Application"
+          visible={displayModal}
+          onOk={handleOk}
+          onCancel={handleCancel}
+          afterClose={handleCancel}
+          className="modalStyle"
+          footer={null}
+        >
+          <h3>{`${currentApplication.first_name} ${currentApplication.last_name}`}</h3>
+          {currentApplication.role_name === 'mentee' ? (
+            <div id="menteeModal">
+              <p>
+                <b>Email:</b> {currentApplication.email}
+              </p>
+              <p>
+                <b>Location:</b> {currentApplication.city},{' '}
+                {currentApplication.state} {currentApplication.country}
+              </p>
+
+              <p>
+                <b>Experience Level:</b> {currentApplication.experience_level}
+              </p>
+              <ul>
+                <b>Membership Criteria:</b>
+                {currentApplication.formerly_incarcerated === true ? (
+                  <li>Formerly Incarcerated</li>
+                ) : null}
+                {currentApplication.low_income === true ? (
+                  <li>Low Income</li>
+                ) : null}
+                {currentApplication.underrepresented_group === true ? (
+                  <li>Belongs to underrepresented group</li>
+                ) : null}
+              </ul>
+              <p>
+                {' '}
+                <b>Convictions:</b>{' '}
+                {`${
+                  currentApplication.formerly_incarcerated === true
+                    ? currentApplication.list_convictions
+                    : 'none'
+                }`}
+              </p>
+              <ul>
+                <b>Applicant needs help with:</b>{' '}
+                {currentApplication.industry_knowledge === true ? (
+                  <li>Industry Knowledge</li>
+                ) : null}
+                {currentApplication.pair_programming === true ? (
+                  <li>Pair Programming</li>
+                ) : null}
+                {currentApplication.job_help === true ? (
+                  <li>Job Help</li>
```

Comment:
What do we think about changing this to 'Job Search Help' or 'Career Readiness Help' or something similar? 'Job Help' could either mean help with your current job OR help getting hired, and clarity is always a good thing. Just a suggestion, not a change we need to make straightaway.